examples/windows_launch_templates/README.md

```diff
@@ -0,0 +1,65 @@
+# Launch templates example
+
+This is EKS example using workers launch template with worker groups feature.
```

Comment:
Please update that this require executing steps from https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#macos-and-linux because windows nodes will not join cluster